packages/app/src/app/api/prices/route.ts

```diff
@@ -0,0 +1,31 @@
+import { NextResponse } from 'next/server';
+
+const COINGECKO_API_URL = 'https://api.coingecko.com/api/v3/simple/price';
+export async function GET(request: Request) {
+  try {
+    const { searchParams } = new URL(request.url);
+    const ids = searchParams.get('ids');
+    const url = `${COINGECKO_API_URL}?ids=${ids}&vs_currencies=usd`;
+    console.log('url', url);
+    const response = await fetch(url, {
+      headers: {
+        'x-cg-demo-api-key': process.env.CMC_API_KEY!,
```

Comment:
do not need key any more for coingecko free plan, and there is wrong header key for coingecko